### PR TITLE
doc: add entry for recent dos vuln

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@sam-github

File: vuln/core/43.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "cve": [
+    "CVE-2017-14919"
+  ],
+  "vulnerable": "4.8.2 || 6.10.2  || 8.x",
```
Comment (sam-github) [inline vuln/core/43.json]: `^4.8.2 || ^6.10.2`, right?

**Comment by mhdawson** [comment]
Pushed update to address comments.

**Comment by mhdawson** [comment]
@sam-github look ok now ?

**Comment by mhdawson** [comment]
Landed as 6d1363395edc82abbb15d4bbf05af72d970ed4b2

**Pull request closed without merging**